src/pages/Post/PostAdd/PostAdd.scss

```
@@ -0,0 +1,88 @@
+@import '../../../style/variables.scss';
+@mixin inputBtn {
+  width: 150px;
+  height: 56px;
+  border-radius: 6px;
+  font-size: 18px;
+  font-weight: 800;
+}
+article {
+  .container {
+    height: 100%;
+    display: flex;
+    justify-content: space-between;
+    padding-bottom: 24px;
+
+    .profileImg {
+      width: 48px;
+      height: 48px;
+      border-radius: 100%;
+      overflow: hidden;
+      border: 1px solid $grey88;
+
+      img {
+        height: 48px;
+      }
+    }
+
+    .content {
+      width: 456px;
+      margin-left: 24px;
+      padding-bottom: 32px;
+
+      .userName {
+        font-weight: bold;
+        color: $black;
+        font-size: 20px;
+        margin-bottom: 8px;
+      }
+      textarea {
+        width: 100%;
+        height: 100%;
+        padding: 15px 16px;
+        resize: none;
+        border: 1px solid $grey88;
+        border-radius: 6px;
+        & ::placeholder {
+          color: $grey60;
+        }
+        & :focus {
+          color: $black;
+        }
+      }
+    }
+  }
+
+  .bottom {
+    background-color: $white;
+    width: 528px;
+    border-top: 1px solid #e6e6e6;
+    padding-top: 24px;
+    .bottomBtn {
+      display: flex;
+      justify-content: space-between;
+      .addCancel {
+        @include inputBtn;
```

선택자 간에는 한 줄씩 띄워서 가독성을 높여주세요!
```suggestion
  .bottom {
    background-color: $white;
    width: 528px;
    border-top: 1px solid #e6e6e6;
    padding-top: 24px;
    
    .bottomBtn {
      display: flex;
      justify-content: space-between;
      
      .addCancel {
        @include inputBtn;
```